docs/migration_guides/index.rst

```diff
@@ -0,0 +1,8 @@
+=======================
+Qiskit Migration Guides
+=======================
```

Comment:
For consistency, can you please use the following characters for title and sections?

```
##########
Title name
##########

Section name
============

Subsection name
---------------

Subsubsection name
^^^^^^^^^^^^^^^^^^

Paragraph name
""""""""""""""
```